liftwizard-logging/liftwizard-config-logging-logstash-encoder/src/main/java/io/liftwizard/dropwizard/configuration/logging/logstash/LogstashEncoderFactory.java

```diff
@@ -318,7 +323,10 @@ private ShortenedThrowableConverter getThrowableConverter()
         {
             throwableConverter.setRootCauseFirst(true);
         }
+        /*
+        TODO: re-enable this after upgrading to logstash-logback-encoder 7.3
         this.truncateStackTracesAfterPrefixes.forEach(throwableConverter::addTruncateAfter);
```

Comment:
@tifanytong the method ThrowableConverter.addTruncateAfter doesn't exist until 7.3.